content/clients/_index.md

```diff
@@ -51,16 +52,16 @@ justify = "center"
     <div class="card">
         <img src="/images/clients/windows.svg"/>
         <div class="bannerbox">
-            <p class="banner">Windows</p>
-            <p class="subbanner">Official</p>
+            <p class="banner">Windows 10</p>
+            <p class="subbanner">Universal Windows Platform</p>
```

Comment:
To go with "respecting the platform holders' guidelines and naming convention, I woud replace this with "Windows Store", since UWP is really effectively a set of libraries and APIs, and not a platform.

Reply:
Doesn't this lead to inconsistencies now, since this row previously announced Official vs 3rd Party?

Reply:
Thinking about it, it's also kind of repeating itself, since "Windows 10" would imply "Windows Store" or "UWP".

I think we should remove the subbaner entirely, if we're going this route.

Reply:
@MrTimscampi Hmm, good point, however, the problem is that in Windows Store there are also Win32 apps ported using Westminster (which is not the case of our native UWP one)

@AndreasGB This client is going to be official.

Reply:
Sure, but what I mean is: The second line is being used in all other cases to proclaim whether a client is official or not, and I would recommend not breaking with this.

Reply:
@AndreasGB Part of my PR was to make a clear division between the kind of client. Now the headers of each section specifies the kind of clients, as it's been discussed here that it's repetitive and unnecessary to repeat "Official" msotly everywhere.

Reply:
Ah, that makes sense then :)